src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,45 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+  min-height: $calendar-height - $cell-size;
+  max-height: $calendar-height;
+  gap: 1px;
+ 
+  @for $index from 1 through 31 {
+    &__cell {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      box-sizing: border-box;
+      width: 100px;
+      height: 100px;
+      background-color: #eee;
+      border: 1px solid black;
+      transition-duration: 0.5s;
```

Comment:
It is better to specify the transition-property too.
Now it applies to all the properties, which is not good for performance
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x